client/src/views/statistics/statistics.ts

```diff
@@ -1,9 +1,102 @@
 import type { Observer } from '@/core/ui/observer';
+import makePercent from '@/core/utils/makePercent';
```

Comment:
```suggestion
import { makePercent } from '@/core/utils/functions';
```

export default 하지 않고 functions.ts로 만들어서 `export function makePercent`하면 좀더 낫지 않을까 합니다.

Reply:
@TsooranKim 

흠 지금 함수가 하나라서 [린트에러](https://github.com/import-js/eslint-plugin-import/blob/v2.23.4/docs/rules/prefer-default-export.md)가 뜨는데 린트에러를 끄고 적용할까요?